src/main/webapp/WEB-INF/tags/admin/search/emailFormFields.tag

```diff
@@ -0,0 +1,20 @@
+<%@ tag description="Admin Search - Form to open email application with default values" %>
+<%@ attribute name="student" type="teammates.ui.template.AdminSearchStudentRow" required="true" %>
+<%@ attribute name="linkTitle" required="true" %>
+<%@ attribute name="subjectType" required="true" %>
+<%@ attribute name="relatedLink" required="true" %>
+<%@ attribute name="sessionStatus" %>
+
+<form class="openEmailApplicationDefaultValues">
```

Comment:
As the one opening #7931 I think you would know that class names should be `like-this`.

Reply:
Done